src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -71,10 +69,7 @@ public ArrayList(int capacity)
             if (capacity < 0) throw new ArgumentOutOfRangeException("capacity", SR.Format(SR.ArgumentOutOfRange_MustBeNonNegNum, "capacity"));
             Contract.EndContractBlock();
 
-            if (capacity == 0)
-                _items = Array.Empty<Object>();
-            else
-                _items = new Object[capacity];
+            _items = capacity == 0 ? Array.Empty<Object>() : new Object[capacity];
```

Comment:
Why make this change? I think it is less readable in this form and also unnecessarily churns the code.